src/kv/kvtypes.h

```diff
@@ -61,21 +61,35 @@ namespace kv
   class TxHistory
   {
   public:
-    using RequestID = std::tuple<size_t /* Caller ID */, size_t /* Client Session ID */, kv::Version>;
+    using RequestID = std::tuple<
```

Comment:
(Not really part of this PR but) This should probably be a struct? Then we only need a std::hash for this type, not for all tuples.

Reply:
We don't hash it at the moment, and this was tweaked somewhat, but now that it's agreed on, yes!

Reply:
I can add this functionality in the next PR where I want to pass the request id into client proxy anyway

Reply:
Yes, that's fine